app/views/sponsors/index.html.haml

```diff
@@ -1,25 +1,20 @@
 .stripe.reverse
   .row
-    .large-12.columns
+    .col.col-lg-9
       %h2 Sponsors
       %p.lead
         Without our sponsors running <strong>codebar</strong> would be impossible. They provide us with space, food and drinks, internet, and they also help us cover our running costs.
       %p.lead
         Thank you for sharing our dream of creating a more welcoming, equal and diverse tech industry.
-
-      %p
+      %p.mb-0
         If you are interested in becoming a host please do get in touch with us at #{mail_to "hello@codebar.io", "hello@codebar.io"}
         ='.'
 
 - ['gold', 'silver', 'bronze', 'community', 'standard'].each do |level|
   - if @sponsor_levels[level]
     .stripe.reverse
       .row
-        .large-12.columns
-          %h3.sponsor-header-margin.text-center
+        .col
+          %h3.text-center
             = sponsorship_level_title(level)
-      - @sponsor_levels[level].in_groups_of(6) do |grouped_sponsors|
-        .row.sponsor-row
-          - grouped_sponsors.compact.each do |sponsor|
-            .medium-2.small-4.columns.sponsor-image
-              = link_to image_tag(sponsor.avatar, class: "small-image", alt: sponsor.name), sponsor.website
+      = render partial: 'shared/sponsors', object: @sponsor_levels[level].compact
```

Comment:
Nice spot! If you want to take this one step further, I noticed that `app/views/chapter/show.html.haml` could br DRYed up as well using this partial…

Reply:
@gnclmorais yep, it's coming up in a future PR

Reply:
![:happy_dance:](https://media.tenor.com/images/08ab7e55e92f352da591c3229a8acb5a/tenor.gif)